### PR TITLE
Update to `system_builder`

### PR DESCRIPTION
**Pull request by daico007**
This PR will:
1. Add support for `BenzeneUA` and `EthanolAA` 
2. Add way to scale up box size (based on suggestion by @jennyfothergill)

I will try to have this all done and ready for review by this Friday.

### CODE REVIEW

**Comment by jennyfothergill** [comment]
@daico007 I added the scaling in #60, so I could start running sims. Although I think your method of providing scaling factors for each box individually is better. :)

File: reproducibility_project/src/molecules/system_builder.py [hunk 1]
```diff
@@ -30,9 +32,10 @@ def construct_system(sp, scale=1.0):
          "forcefield_name": str,
          "cutoff_style": str,
          "r_cut": float (in nm)}
-    scale : float, default 1.0
-        Scale factor by which to scale the box. Useful for system initialization
-        if a shrink step makes equilibration easier.
+    scale_liq_box: float, optional, default=1.0
+        Option to scale sizes of the liquid box.
+    scale_vap_box: float, optional, default=1.0
+        Option to scaple sizes of the vapor box.
```
Comment (justinGilmer) [inline reproducibility_project/src/molecules/system_builder.py:38]:
```suggestion
        Option to scale sizes of the vapor box.
```

File: reproducibility_project/tests/base_test.py [hunk 2]
```diff
@@ -35,6 +35,13 @@ def tmp_job(self, tmp_project, statepoint={"a": 0}):
     def trappe_ua(self):
         return foyer.forcefields.load_TRAPPE_UA()
 
+    @pytest.fixture
+    def benzene_uaff(self):
```
Comment (justinGilmer) [inline reproducibility_project/tests/base_test.py:39]:
```suggestion
    def benzene_ua_ff(self):
```

File: reproducibility_project/tests/test_benzene_ua.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    def test_creation(self):
+        benz = BenzeneUA()
+        assert isinstance(benz, mb.Compound)
+        assert benz.name == "BenzeneUA"
+        assert benz.n_particles == 6
+        assert benz.n_bonds == 6
+        assert [p.name == "_CH" for p in benz.particles]
+        assert len(benz.labels) == 1
+
+    def test_paramters(self, benzene_uaff):
```
Comment (justinGilmer) [inline reproducibility_project/tests/test_benzene_ua.py:20]:
```suggestion
    def test_parameters(self, benzene_ua_ff):
```

File: reproducibility_project/tests/test_benzene_ua.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        nonbonded = benzene_uaff.get_parameters("atoms", key="CH_sp2")
+        bond = benzene_uaff.get_parameters(
+            "harmonic_bonds", key=["CH_E"] * 2, keys_are_atom_classes=True
+        )
+        angle = benzene_uaff.get_parameters(
+            "harmonic_angles", key=["CH_E"] * 3, keys_are_atom_classes=True
+        )
+        rbtorsion = benzene_uaff.get_parameters(
+            "rb_propers", key=["CH_E"] * 4, keys_are_atom_classes=True
+        )
```
Comment (justinGilmer) [inline reproducibility_project/tests/test_benzene_ua.py:32]:
```suggestion
        param_struct = benzene_ua_ff.apply(BenzeneUA)

        nonbonded = benzene_ua_ff.get_parameters("atoms", key="CH_sp2")
        bond = benzene_ua_ff.get_parameters(
            "harmonic_bonds", key=["CH_E"] * 2, keys_are_atom_classes=True
        )
        angle = benzene_ua_ff.get_parameters(
            "harmonic_angles", key=["CH_E"] * 3, keys_are_atom_classes=True
        )
        rbtorsion = benzene_ua_ff.get_parameters(
            "rb_propers", key=["CH_E"] * 4, keys_are_atom_classes=True
        )
```

File: reproducibility_project/tests/test_ethanol_aa.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
+import foyer
+import mbuild as mb
+
+from reproducibility_project.src.molecules.ethanol_aa import EthanolAA
+from reproducibility_project.tests.base_test import BaseTest
+
+
+class TestEthanolAA(BaseTest):
+    """Test to ensure BenzeneUA behaves as expected."""
```
Comment (justinGilmer) [inline reproducibility_project/tests/test_ethanol_aa.py:9]:
```suggestion
    """Test to ensure EthanolAA behaves as expected."""
```

**Review by justinGilmer — CHANGES_REQUESTED** [review]
Small changes then LGTM!

File: reproducibility_project/src/molecules/pentane_ua.py [hunk 6]
```diff
@@ -5,7 +5,7 @@
 
 
 class PentaneUA(mb.Compound):
-    """Create a single particle methane compound."""
+    """Create a single particle pentane compound."""
```
Comment (jennyfothergill) [inline reproducibility_project/src/molecules/pentane_ua.py:8]:
```suggestion
    """Create a united-atom pentane compound."""
```

**Review by jennyfothergill — APPROVED** [review]
minor change, but otherwise LGTM!

**Review by justinGilmer — APPROVED** [review]
LGTM!

**Pull request merged**